testing/run-scripts/run_cloud.sh

```diff
@@ -48,6 +50,7 @@ while getopts p:z:s:i:uwd:b:ah? opt; do
     d) PUBLIC_IP="$OPTARG" ;;
     b) BANNER="$OPTARG" ;;
     a) AUTOMATED=true ;;
+    k) KEY="$OPTARG" ;;
```

Comment:
Let's a check in `if [ "$UPDATE" = true ]` block for `-k` being specified - that should raise an error. See line 89.